lib/active_model_serializers/test/serializer.rb

```diff
@@ -0,0 +1,125 @@
+require 'set'
+module ActiveModelSerializers
+  module Test
+    module Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+        teardown :teardown_serialization_subscriptions
```

Comment:
I don't think it needs a test. 

 It just unsubscribes the subscriber created in the test...

 See more in the commit https://github.com/rails-api/active_model_serializers/commit/d9eefdba7111909bedfc85766fd1d05fe191cdc3


I can try to show what may break without it, but given that the teardown is copied from rails, and I understand the behavior with and without...

B mobile phone

> On Jan 5, 2016, at 1:05 PM, Mauro George <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/test/serializer.rb:
> 
> > @@ -0,0 +1,125 @@
> > +require 'set'
> > +module ActiveModelSerializers
> > +  module Test
> > +    module Serializer
> > +      extend ActiveSupport::Concern
> > +
> > +      included do
> > +        setup :setup_serialization_subscriptions
> > +        teardown :teardown_serialization_subscriptions
> We really need this teardown ? If we need, I think we need to write a broken test to it, what behavior will be broken if we do not have this teardown.
> 
> —
> Reply to this email directly or view it on GitHub.
>